packages/core/test/toast/overlayToasterTests.tsx

```diff
@@ -16,346 +16,281 @@
 
 import { waitFor } from "@testing-library/dom";
 import { assert } from "chai";
-import { mount } from "enzyme";
-import * as React from "react";
 import * as ReactDOMClient from "react-dom/client";
 import sinon, { spy } from "sinon";
 
 import { expectPropValidationError } from "@blueprintjs/test-commons";
 
-import { Classes, OverlayToaster, type OverlayToasterProps, type Toaster } from "../../src";
-import { TOASTER_CREATE_NULL, TOASTER_MAX_TOASTS_INVALID } from "../../src/common/errors";
-import { OVERLAY_TOASTER_DELAY_MS } from "../../src/components/toast/overlayToaster";
+import { Classes, OverlayToaster, type Toaster } from "../../src";
+import { TOASTER_MAX_TOASTS_INVALID } from "../../src/common/errors";
+import { OVERLAY_TOASTER_DELAY_MS, type OverlayToasterDOMRenderer } from "../../src/components/toast/overlayToaster";
```

Comment:
The diff in this file isn't actually so big, its mostly whitespace changes. Recommend enabling the "hide whitespace" option in github.